insights/tools/insights_inspect.py

```diff
@@ -43,7 +43,7 @@
 
 >>> insights-inspect insights.specs.Specs.hostname
 IPython Console Usage Info:
-Enter 'make_pass.' and tab to get a list of properties
+Enter 'Hostname.' and tab to get a list of properties
```

Comment:
This should actually be ...
Enter 'hostname.'
Line 58 below should be 'hostname.cmd' instead of 'CommandOutputProvider.cmd'